cadc-haproxy-dev/src/haproxy/cacerts/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This should be renamed to cadc-internal-ca.crt in the repo

Reply:
I can do that but then rsync-ing with the servers won't be as simple.